src/encoded/types/access_key.py

```diff
@@ -18,4 +27,39 @@
     },)
 class AccessKey(SMAHTItem, SnovaultAccessKey):
     """AccessKey class."""
+    ACCESS_KEY_EXPIRATION_TIME = 90  # days
+    item_type = 'access_key'
+    name_key = 'access_key_id'
     schema = mixin_smaht_permission_types(SNOVAULT_ACCESS_KEY_SCHEMA)
+    embedded_list = []
+
+    STATUS_ACL = {
+        'current': [(Allow, 'role.owner', ['view', 'edit'])] + ONLY_ADMIN_VIEW_ACL,
+        'deleted': DELETED_ACL,
+    }
+
+    class Collection(SMAHTItem.Collection):
+        pass
+
+
+@view_config(context=AccessKey.Collection, request_method='POST',
+             permission='add',
+             validators=[validate_item_content_post])
+@debug_log
+def access_key_add(context, request):
+    return sno_access_key_add(context, request)
```

Comment:
Guessing this has to be done this way rather than by just using include from snovault so it goes through the right permissioning?